### PR TITLE
Validation form

### PR DESCRIPTION
**Pull request by ezraermy**
This is a pull request. I:
- created branches with different base branches.
- Followed GitHub flow.
- Added a descriptive and easy to understand README
- Edited HTML file
- Edited CSS file
- Edited JavaScript file
- Added form validation

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -159,3 +159,19 @@ menuItems.forEach(
     menuItem.addEventListener('click', toggleMenu);
   },
 );
+
+const email = document.querySelector('#email');
+const form = document.querySelector('.my-form');
+const error = email.nextElementSibling;
+const pattern = /^[a-z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-z0-9-]+(?:\.[a-z0-9-]+)*$/;
```
Comment (Kossi-stack) [inline js/index.js:166]: -[ ] Kindly make sure when a user submits the contact form with the email field containing uppercase characters an error message should show up. I submitted the email field with uppercase and it went through kindly fix it.

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @ezraermy @Iqbal-Elham,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: js/index.js (hunk 1 above)
Comment (ezraermy) [reply js/index.js:166]: HI Kossi,

Thank you very much for your kind feedback. However, without changing anything I've tried it on my pc. The result is as shown bellow fig.
validation_form

**Review by Santiago220991 — APPROVED** [review]
Hi @ezraermy, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
:heavy_check_mark: Applied changes suggested by the last reviewer.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Followed GitHub flow.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**